iroha/src/sumeragi/mod.rs

```diff
@@ -767,6 +823,30 @@ impl<Q: QueueTrait, G: GenesisNetworkTrait, W: WorldTrait> Sumeragi<Q, G, W> {
 
         self.topology.clone()
     }
+
+    /// Connects all peers from current network topology.
+    pub async fn connect_peers(&self) {
+        // The most needed delay!!!
+        if self.peer_connections_delay > 0 {
+            let delay: u64 = rand::random();
+            tokio::time::sleep(Duration::from_millis(delay % self.peer_connections_delay)).await;
+        }
```

Comment:
Yes. Otherwise peers just connect to each other in 1ms, and both think that their connect is a tween, and close it.
But I'm trying to change that, maybe we will remove it in the end.

Reply:
Dont like idea of sleeps with random time. Not in the context of problem though

Reply:
Removed.